src/odemis/dataio/test/multi_tiff_test.py

```diff
@@ -0,0 +1,102 @@
+
+import datetime
+import glob
+import json
+import logging 
+import os
+import unittest
+
+import numpy
+
+from odemis import model
+from odemis.dataio import tiff
+from odemis.util.dataio import open_acquisition
+
+logging.basicConfig(level=logging.DEBUG)
+
+# write test metadata to file
+def write_test_metadata(path: str):
+
+    filenames =  glob.glob(os.path.join(path, "**/*.tif"), recursive=True) 
+    test_metadata = {}
+
+    for fname in filenames:
+
+        try:
+            data = open_acquisition(fname)[0].getData()
+
+            # format metadata for tests
+            tmd = data.metadata
+            tmd["filename"] = os.path.basename(fname).replace(".tif", "")
+            tmd["shape"] = data.shape
+            tmd["dtype"] = str(data.dtype)
+            tmd["length"] = 1
+
+            # convert timestamp into datetime
+            ts = data.metadata[model.MD_ACQ_DATE]
+            tmd["timestamp"] = ts
+            tmd["datetime"] = datetime.datetime.fromtimestamp(ts).strftime('%Y-%m-%d %H:%M:%S')
+
+            test_metadata[tmd["filename"]] = tmd
+
+        except:
+            continue # ignore files that cannot be read for now
+
+    # save test_metadata as json
+    import json
+    with open(os.path.join(path, "test_metadata.json"), "w") as f:
+        json.dump(test_metadata, f)
+    
+    return test_metadata
+
+
+class TestMultiTiffIO(unittest.TestCase):
+
+  def test_manufacturer_metadata_parsers(self):
+    """Test open_data function with multiple manufacturers and metadata parsers."""
+
+    # test data path (only available on hosted runner atm)
+    TEST_PATH = os.path.join(f"{os.path.expanduser('~')}", "test_data/meteor/test_images")
+    IGNORED_KEYS = ["shape", "dtype", "length", "filename", "timestamp", "datetime"]
+
+    # if test path doesnt exist, we are on github actions, just ignore for now
+    # TODO: download test data for github actions
+    if not os.path.exists(TEST_PATH):
+        logging.warning(f"Test data not found at {TEST_PATH}. Skipping test.")
+        return
```

Comment:
You can also skip the test, then it will show up as skipped in the report, instead of as passed.
```suggestion
        raise unittest.SkipTest(f"Test data not found at {TEST_PATH}. Skipping test.")
```